src/test/java/test/asserttests/AssertTest.java

```diff
@@ -149,4 +149,33 @@ private static ThrowingRunnable throwingRunnable(final Throwable t) {
   public void doubleNaNAssertion() {
     Assert.assertEquals(Double.NaN, Double.NaN, 0.0);
   }
+
+  @Test
+  public void arrayWithNullValues_1(){
```

Comment:
Can we please help club all these 4 methods into just 1 data driven test which tests out all these conditions ?

Reply:
Sure, I will create data driven tests for them.